tests/page-objects/apdex/contacts.page.js

```diff
@@ -34,9 +39,10 @@ class ContactsPage extends Page {
     await super.fillUpForm(form, commonElements);
   }
 
-  async searchPatient(settingsProvider) {
-    const form = settingsProvider.getForm('patientSearch');
-    await super.searchContact(form);
+  async searchContact(settingsProvider) {
```

Comment:
Renaming from `searchPatient` to `searchContact`, since technically, anyone can configure the tests to search by any household, facility, or patient.